plugins/acss-kit/commands/kit-add-html.md

```
@@ -0,0 +1,57 @@
+description: Generate static HTML versions of fpkit-style components (markup + SCSS + optional vanilla JS) for non-React projects
+argument-hint: <component> [component2 ...]
+allowed-tools: Read, Glob, Grep, Write, Edit, Bash, AskUserQuestion
+---
+
+# /kit-add-html
+
+Generate **static HTML** versions of fpkit-style components for projects that don't use React — server-rendered apps, static sites, design-system docs, email templates, prototypes. The same component reference docs power both `/kit-add` (React TSX) and this command (HTML).
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)